_modules/Week 5 -  Digging into a digital common : the case of Decidim

```diff
@@ -0,0 +1,10 @@
+title: Week 5 -  Digging into a digital common : the case of Decidim
+---
+
+Slides
+: **PDF**{:  }
+  : [Download](/assets/pdfs/Digital%20Commons%20and%20Public%20Policy%20-%20Week%201.pdf)
+
+Bibliography
+: **VIDEO**{:}
```

Comment:
If you want the badges to display like this
<img width="143" alt="Capture d’écran 2023-09-26 à 00 43 54" src="https://github.com/virgile-dev/digital-commons-and-public-policy/assets/11473995/34cf1423-c29a-4115-ab6a-a1b3894cddf0">
You need to have between the `{}` this `: .label .label-purple`